src/library/export/coverartcopywizard.h

```diff
@@ -0,0 +1,36 @@
+// CoverArtCopyWizard handles copying the cover art of the track to a where track located.
+
+#pragma once
+
+#include <QScopedPointer>
+#include <QString>
+
+#include "library/export/coverartcopyworker.h"
+#include "library/export/dlgcoverartcopy.h"
+#include "preferences/usersettings.h"
+
+class CoverArtCopyWizard : public QObject {
+    Q_OBJECT
+  public:
+    CoverArtCopyWizard(QWidget* parent,
+            UserSettingsPointer pConfig,
+            const QImage& coverArtImage,
+            const QString& coverArtCopyPath)
+            : m_parent(parent),
+              m_pConfig(pConfig),
+              m_coverArtImage(coverArtImage),
+              m_coverArtCopyPath(coverArtCopyPath) {
+    }
+    virtual ~CoverArtCopyWizard() {
+    }
```

Comment:
The function body can be replaced with `= default;`

Reply:
Thank you, I must've forgotten that.